web-ng/src/app/components/map/map.component.ts

```diff
@@ -94,20 +106,32 @@ export class MapComponent implements OnInit, AfterViewInit {
         };
       })
     );
+
+    this.drawingKitService.getEditMode$().subscribe(editMode => {
```

Comment:
Please save the subscription and dispose in `ngOnDestroy()` to prevent memory leaks.